src/main/java/org/wildfly/security/http/HttpExchangeSpi.java

```diff
@@ -77,6 +81,29 @@ default SSLSession getSSLSession() {
     }
 
     /**
+     * Get the peer certificates (if any) associated with the current connection.
+     *
+     * Although re-negotiation may be requested the underlying implementation may not support re-negotiation and will continue
+     * to return {@code null}. {@code null} will also still be returned if after re-negotiation peer certificates are still not
+     * available.
+     *
+     * @param renegotiate if no certificates are available should re-negotiation of the session be attempted.
+     * @return the peer certificates associated with the current connection or {@code null} if none associated.
+     */
+    default Certificate[] getPeerCertificates(boolean renegotiate) {
```

Comment:
Yes the default implementation does not support renegotiation so ignores the parameter.